cmd/main.go

```diff
@@ -88,7 +88,7 @@ func run(config *config.Config) {
 
 	klog.Infof("ZFS Driver Version :- %s - commit :- %s", version.Current(), version.GetGitCommit())
 	klog.Infof(
-		"DriverName: %s Plugin: %s EndPoint: %s NodeID: %s",
+		"DriverName: %s Plugin: %s EndPoint: %s Node Name: %s",
```

Comment:
`NodeID` is used [here](https://github.com/openebs/zfs-localpv/pull/304/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R95)

Reply:
it is config.NodeID, which is node name only.